.circleci/images/node-8/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM node:8
+RUN apt-get install fonts-dejavu-core
```

Comment:
Since t his is shared between both Dockerfiles, it might be worth pulling out into a shell script (in case we want to add more complex stuff) and calling the script from both.

Reply:
Actually I wonder whether this is worth layering the other way around: A base Shields layer with all the stuff that's required everywhere (like `fonts-dejavu-core`), then layers on top of that that install the right Node.js version. That means we couldn't use the nice base `node:6` and `node:8` layers though. Hmm.

Reply:
Hmm, I'm not sure it would speed things up that much. I'd expect our layer to be the smaller part. And yea, there's a lot of work that goes into those base node images. I'm glad we can leverage it.